demos/serverless/src/handlers.js

```diff
@@ -127,7 +127,13 @@ exports.logs = async (event, context) => {
     });
   }
   putLogEventsInput.logEvents = logEvents;
-  await cloudWatchClient.putLogEvents(putLogEventsInput).promise();
+  await cloudWatchClient.putLogEvents(putLogEventsInput).promise().then(() => {
```

Comment:
Since we are already using `promise()`, recommend using the `try/catch` block instead of callbacks.

```
try {
  await cloudWatchClient.putLogEvents(putLogEventsInput).promise();
  console.log(...);
} catch (error) {
  console.log(...);
}
console.log(putLogEventsInput);
```